lxd-benchmark/main.go

```diff
@@ -33,7 +33,7 @@ func main() {
 
 func run(args []string) error {
 	// Parse command line
-	if len(os.Args) == 1 || !shared.StringInSlice(os.Args[1], []string{"spawn", "start", "stop", "delete"}) {
+	if len(os.Args) == 1 || !shared.StringInSlice(os.Args[1], []string{"launch", "start", "stop", "delete"}) {
```

Comment:
I'm not opposed to this, but you need to add code so that if os.Args[1] is "spawn", you rewrite it to "launch", otherwise this is breaking backward compatibility.